### PR TITLE
docs: update some code groups with package manager examples

### PR DESCRIPTION
**Pull request by rexhent**
### 🔗 Linked issue

Fixes on instance of #27790

### 📚 Description

Improves documentation for users not using npm by allowing them to copy and past the commands from the upgrade page using their preferred package manager.

### CODE REVIEW

File: docs/1.getting-started/12.upgrade.md [hunk 1]
```diff
@@ -11,10 +11,22 @@ navigation.icon: i-ph-arrow-circle-up-duotone
 
 To upgrade Nuxt to the [latest release](https://github.com/nuxt/nuxt/releases), use the `nuxi upgrade` command.
 
-```bash [Terminal]
+::code-group
+
+```bash [npx]
 npx nuxi upgrade
 ```
```
Comment (GalacticHypernova) [inline docs/1.getting-started/12.upgrade.md:19]: Might also add yarn in here

Comment (rexhent) [reply docs/1.getting-started/12.upgrade.md:19]: Sure. That would be yarn dlx right?

File: docs/1.getting-started/12.upgrade.md [hunk 2]
```diff
@@ -11,10 +11,22 @@ navigation.icon: i-ph-arrow-circle-up-duotone
...
 
+```bash [pnpm]
+pnpm dlx nuxi upgrade
```
Comment (manniL) [inline docs/1.getting-started/12.upgrade.md:21]:
```suggestion
pnpx nuxi upgrade
```

Should be a shorthand IIRC

File: docs/1.getting-started/12.upgrade.md (hunk 1 above)
Comment (GalacticHypernova) [reply docs/1.getting-started/12.upgrade.md:19]: Indeed it is

File: docs/1.getting-started/12.upgrade.md (hunk 2 above)
Comment (MuhammadM1998) [reply docs/1.getting-started/12.upgrade.md:21]: `pnpx` is deprecated now and `pnpm dlx` is the right command (unfortunately)

Comment (manniL) [reply docs/1.getting-started/12.upgrade.md:21]: Didn't know - apparently so is `npx` instead of `npm exec` (or they plan to)

Comment (manniL) [reply docs/1.getting-started/12.upgrade.md:21]: Sure, why not 👍🏻

(Don't have a strong opinion about bunx or bun x tbh)

Comment (rexhent) [reply docs/1.getting-started/12.upgrade.md:21]: > Sure, why not 👍🏻
> 
> (Don't have a strong opinion about bunx or bun x tbh)

I'm having some trouble running locally is it okay to ask you, and if so is this an appropriate place?
In both my laptop and my pc at home. I've set the nuxt.com docs in the env to be the docs folder of my nuxt repo, but I'm still getting some errors.

Comment (rexhent) [reply docs/1.getting-started/12.upgrade.md:24]: I got it working on my linux machine

> > Sure, why not 👍🏻
> > (Don't have a strong opinion about bunx or bun x tbh)
> 
> I'm having some trouble running locally is it okay to ask you, and if so is this an appropriate place? In both my laptop and my pc at home. I've set the nuxt.com docs in the env to be the docs folder of my nuxt repo, but I'm still getting some errors.

So what would be more appropriate, just have this pr for upgrade.md? Or have all the changes in the one PR?

Comment (manniL) [reply docs/1.getting-started/12.upgrade.md:24]: Change it everywhere if you want to ☺️

**Comment by rexhent** [comment]
I just noticed my prettier config went and changed alot of stuff

**Comment by JakeIsMeh** [comment]
Perhaps it'd be better to use the [package manager components from UnDocs](https://undocs.pages.dev/guide/components#package-manager)?

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you! ❤️

**Pull request merged**